include/vrv/scoringupfunctor.h

```diff
@@ -107,6 +107,7 @@ class ScoringUpFunctor : public Functor {
     int m_modusMinor;
     int m_tempus;
     int m_prolatio;
+    std::tuple<int, int, int, int> m_mensurAsTuplet;
```

Comment:
Why not use a `Mensur` object? That would avoid all the `tuple` hassle. See [this](https://github.com/rism-digital/verovio/blob/a31c9f6ce10b912bf069c64b930448972fc168b6/src/durationinterface.cpp#L143) as an example

As you will see, the only trick is that you need to use `abs()` when using the values since default is `-3`. So `Mensur::GetProlatio()` will return -3 (when nothing is set) or otherwise 2 or 3. See the definition [here](https://github.com/rism-digital/verovio/blob/a31c9f6ce10b912bf069c64b930448972fc168b6/libmei/addons/attdef.h#L233-L237)